app/webpacker/components/CompetitionResultSubmission/ImportResultsData/index.jsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from 'react';
+import { Accordion, Container, Message } from 'semantic-ui-react';
+import UploadResultsJson from '../UploadResultsJson';
+import WCAQueryClientProvider from '../../../lib/providers/WCAQueryClientProvider';
+
+export default function Wrapper({
+  competitionId,
+  alreadyHasSubmittedResult,
+  isWrtViewing,
+}) {
+  return (
+    <WCAQueryClientProvider>
+      <ImportResultsData
+        competitionId={competitionId}
+        alreadyHasSubmittedResult={alreadyHasSubmittedResult}
+        isWrtViewing={isWrtViewing}
+      />
+    </WCAQueryClientProvider>
+  );
+}
+
+function ImportResultsData({
+  competitionId,
+  alreadyHasSubmittedResult,
+  isWrtViewing,
+}) {
+  const [activeAccordion, setActiveAccordion] = useState(!alreadyHasSubmittedResult);
+
+  return (
+    <Container fluid>
+      <Accordion fluid styled>
+        <Accordion.Title
+          active={activeAccordion}
+          onClick={() => setActiveAccordion((prevValue) => !prevValue)}
+        >
+          Import Results Data
+        </Accordion.Title>
+        <Accordion.Content active={activeAccordion}>
+          {alreadyHasSubmittedResult
+            ? (
+              <Message warning>
+                Some results are already there, importing results data again will override all
+                of them!
+              </Message>
+            )
+            : (
+              <Message info>
+                Please start by selecting a JSON file to import.
+              </Message>
+            )}
```

Comment:
The result is always a `Message` component in any case, so you could do something like:
```jsx
<Message warning={alreadyHasSubmitted} info={!alreadyHasSubmitted}>
  ...
</Message>
```